src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/scalable/LabeledVariantAnnotationsWalker.java

```diff
@@ -283,7 +304,9 @@ VCFHeader constructVCFHeader(final List<String> sortedLabels) {
                 .collect(Collectors.toCollection(TreeSet::new));
         hInfo.add(GATKVCFHeaderLines.getFilterLine(VCFConstants.PASSES_FILTERS_v4));
         final SAMSequenceDictionary sequenceDictionary = getBestAvailableSequenceDictionary();
-        hInfo = VcfUtils.updateHeaderContigLines(hInfo, null, sequenceDictionary, true);
+        if (sequenceDictionary != null) {
```

Comment:
This avoids a complaint raised if the VCF is missing contig lines in the header.